lib/src/Pos/Client/CLI/NodeOptions.hs

```diff
@@ -173,17 +173,18 @@ nodeWithApiArgsParser =
 nodeApiArgsParser :: Parser NodeApiArgs
 nodeApiArgsParser =
     NodeApiArgs
-        <$> addressParser
+        <$> addressParser "node-api-address" (localhost, 8083)
         <*> tlsParamsParser
         <*> debugModeParser
+        <*> addressParser "node-doc-address" (localhost, 8084)
```

Comment:
I think we should be defaulting to `8080` (api) and `8180` (doc) as this is the current default in our wallet backend:

```
    monitoringApiPortParser :: Parser Word16
    monitoringApiPortParser = CLI.webPortOption 8080 "Port for the monitoring API."
```

Reply:
Then there will be a clash for the case when the node and wallet are running on the same host

Reply:
Well, that's okay, we won't expose the monitoring API from the node anymore!